modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -185,7 +185,12 @@ class Editor implements EditorObservable {
   public mode: Mode;
 
   /**
+   * Sets the editor mode. Mode can be for example "design", "code" or "readonly".
```

Comment:
Hm, do we need "for example"? "be for example" always feel like an odd sequence of words to me, especially without commas. I'm probably nitpicking though, especially on something that's been deprecated.

Reply:
This is just literally restoring what was deleted and adding the deprecated note.

Reply:
Removing `for example` seemed to make it worse, so just removed the `Mode can be` instead.